ch15/bias_intro.md

```diff
@@ -7,5 +7,4 @@ fits the noise in the data rather than the data's overall pattern.
 To understand why this happens, we analyze our models using the tools of
 probability and statistics. These tools allow us to generalize beyond a few
 isolated examples to describe fundamental phenomena in modeling. In
-particular, we will use the tools of expectation and variance from chapter
-12 to uncover the bias-variance tradeoff.
+particular, we will use the tools of expectation and variance to uncover the bias-variance tradeoff.
```

Comment:
Thanks for also editing this intro! I appreciate how thorough you are with these edits.